terraform/test/test.tf

```diff
@@ -1,6 +1,9 @@
 ### Provide variables in non-Git tfvars file
 variable "vpc_id" {}
+<<<<<<< HEAD
+=======
 variable "vpc_cidr_block" {}

```

Comment:
looks like you might have a merge error to fix

Reply:
THANK YOU!